### PR TITLE
added support to more than one registry url for hec

### PR DESCRIPTION
**Pull request by nabobalis**
Current reg is down, found another one but decided that we should just check each one.

### CODE REVIEW

File: sunpy/net/helio/parser.py [hunk 1]
```diff
@@ -37,17 +39,21 @@ def webservice_parser(service='HEC'):
...
-    link = RL.LINK + '/' + service.lower()
-    xml = link_test(link)
+    xml = None
+    for REG_LINK in REG_LINKS:
+        link = REG_LINK + service.lower()
+        xml = link_test(link)
+        if xml:
+            break
     if xml is None:
         return None
```
Comment (nabobalis) [inline sunpy/net/helio/parser.py:58]: I feel like this logic could be made simpler and nicer.

**Comment by nabobalis** [comment]
Online failure is the cutout service.

File: sunpy/net/helio/parser.py [hunk 2]
```diff
@@ -175,9 +181,10 @@ def link_test(link):
         None
     """
     try:
-        with closing(urllib.request.urlopen(link, timeout=LINK_TIMEOUT)) as fd:
```
Comment (Cadair) [inline sunpy/net/helio/parser.py:178]: what's closing do?

File: sunpy/net/helio/parser.py (hunk 1 above)
Comment (Cadair) [reply sunpy/net/helio/parser.py:58]: it'll suffice

File: sunpy/net/helio/parser.py (hunk 2 above)
Comment (nabobalis) [reply sunpy/net/helio/parser.py:178]: I assume it was before a time the urlopen was unable to be opened as a context manager.

**Review by Cadair — APPROVED** [review]

File: sunpy/net/helio/parser.py (hunk 2 above)
Comment (Cadair) [reply sunpy/net/helio/parser.py:178]: huuuh. mybbe.

Comment (nabobalis) [reply sunpy/net/helio/parser.py:178]: https://docs.python.org/3/library/contextlib.html#contextlib.closing

File: sunpy/net/helio/parser.py (hunk 1 above)
Comment (nabobalis) [reply sunpy/net/helio/parser.py:58]: I rather do better than, "it'll suffice".

**Comment by nabobalis** [comment]
The doctests failed and returned None, I don't want to merge a PR where I've made the tests more flaky.

**Pull request merged**